care/emr/models/supply_request.py

```
@@ -0,0 +1,33 @@
+from django.db import models
+
+from care.emr.models import EMRBaseModel
+
+
+class SupplyRequest(EMRBaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)